packages/service-worker/src/services/accounts/utils/addAllAccountsWithHistory.ts

```diff
@@ -0,0 +1,60 @@
+import { container } from 'tsyringe';
+import { ModuleManager } from '~/vmModules/ModuleManager';
+import { AccountsService } from '../AccountsService';
+import { NetworkService } from '~/services/network/NetworkService';
+import { AccountType } from '@core/types';
+
+interface CheckAccountsHistoryParams {
+  walletId: string;
+}
+
+export const addAllAccountsWithHistory = async ({
```

Comment:
This function should ensure that at least 1 account is added. Currently new onboardings with new mnemonics or never used wallets are breaking.